cpp-client/build-dependencies.sh

```diff
@@ -530,8 +531,8 @@ if [ "$CLONE_ZLIB" = "yes" ]; then
   echo
   echo "*** Clone zlib"
   cd $SRC
-  # Previously used version: v1.2.11
-  git clone $GIT_FLAGS -b v1.2.13 --depth 1 "${GITHUB_BASE_URL}/madler/zlib"
+  # Previously used version: v1.2.13
+  git clone $GIT_FLAGS -b v1.3 --depth 1 "${GITHUB_BASE_URL}/madler/zlib"
```

Comment:
Do we need to specify the patch version, or are we ok with that changing?

Reply:
That's a specific version, is just the name they gave, in git terms is just some arbitrary string.  Mean as far as I understand how the universe works, that won't change when they create a new version.